lib/js/dom.nim

```diff
@@ -226,19 +232,22 @@ type
     valid*: bool
     valueMissing*: bool
 
-  # https://developer.mozilla.org/en-US/docs/Web/API/Blob
   Blob* = ref BlobObj
   BlobObj {.importc.} = object of RootObj
     size*: int
     `type`*: cstring
 
-  # https://developer.mozilla.org/en-US/docs/Web/API/File
```

Comment:
Why remove these?

Reply:
see https://github.com/nim-lang/Nim/pull/13483#discussion_r383606456 seems like most reviewers agree these comments were valuable; https://github.com/nim-lang/Nim/pull/13483#discussion_r388564243 otherwise would be a good compromise

Reply:
yeah, it's useful to have a reference to what the definition is based on.

Reply:
I have made all comments pointing to documentation be ##doc comments are requested.